modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectActionAPI.macro

```diff
@@ -0,0 +1,48 @@
+definition {
+
+	macro _curlObjectAction {
```

Comment:
why do we need to have it separate from the body?

Reply:
In my opinion, if we have other tests that need to use object action in the future, they can use the __curlObjectAction method as their basic, what do you think of it?

Reply:
ok, I understand, yeah that's a good idea :) However, we can put the body in the same method